### ISSUE TITLE
Google Drive Links are broken 

### ISSUE THREAD

**Issue by gblong1** [zShanCS/openai-autoflow#4]
I am going through the setup, and am trying to download the models in steps 3-5. However the google drive links are all broken, giving a "Page Not Found" error.

Are these files hosted elsewhere? The goal it to use this server with the vs-code-extension project.

**Comment by zShanCS** [comment]
Yes, the embeddings are no longer there. You will have to regenerate them.
Infact, this time you can directly use chatgpt api to do it if thats okay.
We used machine learning algorithms to map a natural language sentence to
one of the autoflow commands. Since chatgpt is very good at it now, you
should simply use that.

Other than that, I am all out of ideas.

Thanks.
